usl_pipeline/usl_lib/usl_lib/transformers/raster_transformers.py

```diff
@@ -0,0 +1,73 @@
+from typing import Tuple
+
+import numpy
+import numpy.typing as npt
+from shapely import geometry  # import Point, Polygon
```

Comment:
What's the `# import Point, Polygon` comment meant to describe?

Reply:
Forgot to remove it.